src/13_file_io.py

```diff
@@ -10,9 +10,17 @@
 
 # YOUR CODE HERE
 
+
+print(open("foo.txt","r").read())
```

Comment:
Also can use a ```with``` statement to open the file (a few advantages of doing it this way):
```python
with open("foo.txt") as f:
    read_data = f.read()
```